docs/README.rst

```diff
@@ -201,67 +213,86 @@ sslProfile           JSON object       Optional                   BIG-IP SSL pro
+==================== ================= ============== ======================================================= ====================================
+partition            string            Required       The BIG-IP partition you want the |kctlr| to manage.                                            
+-------------------- ----------------- -------------- ------------------------------------------------------- ------------------------------------
+iapp                 string            Required       BIG-IP iApp template to use to create the               Any iApp template that already 
+                                                      application  Service.                                   exists on the BIG-IP system. 
+-------------------- ----------------- -------------- ------------------------------------------------------- ------------------------------------
+iappPoolMemberTable  JSON object       Required       Define the name and layout of the pool member table   
+                                                      in the iApp.                                          
+                                    
+                                                      **See** :ref:`iApp Pool Member Table`.         
+-------------------- ----------------- -------------- ------------------------------------------------------- ------------------------------------
+iappTables           JSON object       Optional       Define iApp tables to apply to the Application Service
+                     array
+                                                      Example: ::
+                                                      
+                                                      "iappTables": {                           
+                                                        "monitor__Monitors":                    
+                                                          {"columns": ["Index", "Name", "Type", "Options"], 
+                                                           "rows": [[0, "mon1", "tcp", "" ],                
+                                                                    [1, "mon2", "http", ""]]}}"             
+-------------------- ----------------- -------------- ------------------------------------------------------- ------------------------------------
+iappOptions          key-value object  Required       Define the App configurations                           See :ref:`configuration parameters`.        
+-------------------- ----------------- -------------- ------------------------------------------------------- ------------------------------------
+iappVariables        key-value object  Required       Define the iApp variables needed for Service creation.
+                              
+==================== ================= ============== ======================================================= ====================================
+
+.. _iapp pool member table:
 
 iApp Pool Member Table
```

Comment:
This section should be a subheading of iApps I think

Reply:
fixed